### PR TITLE
tests: 100% test coverage

### PR DESCRIPTION
**Pull request by jacquerie**
ping @jalavik.

I also included my proposals for the API and the configuration, which will be split in another commit.

### CODE REVIEW

**Comment by jalavik** [comment]
:+1:

**Comment by jacquerie** [comment]
@jalavik ready for review.

File: invenio_grobid/api.py [hunk 1]
```diff
@@ -33,16 +33,13 @@
 from .errors import GrobidRequestError
 
 
-def process_pdf(pdf_file, stream=False):
+def process_pdf(pdf_file):
```
Comment (jalavik) [inline invenio_grobid/api.py]: Shall we indicate that it expects a stream object? E.g. `process_pdf_stream` or in the docstring?

Comment (jacquerie) [inline invenio_grobid/api.py]: I like both ideas! Will do.

**Comment by jacquerie** [comment]
@jalavik incorporated changes from review.

**Pull request merged**